### PR TITLE
LIG Read Issue #285

### LINKED ISSUE #285: Terraform crashes for Logical Interconnect Group Read
OV: v6.0.0
terraform-provider-oneview: v6.0.0-12 and v6.0.0-11

Current Situation: Terraform crashed while performing Read due to improper collection of Qos Attributes.

### PR DESCRIPTION
**Pull request by AsisBagga**
### Description
TF Crashes for Read in LIG due to the bug in LIG Read method while collecting Qos attributes. 

### Issues Resolved
#285 

### Check List
- [x] New functionality includes testing.
  - [x]  All tests pass for go 1.11 + gofmt checks.
- [x] New functionality has been documented in the README if applicable.
  - [x] New functionality has been thoroughly documented in the examples (please include helpful comments).
  - [x] New endpoints supported are updated in the endpoints-support.md file.
- [x] Changes are documented in the CHANGELOG.

### CODE REVIEW

File: data_source.tf [hunk 1]
```diff
@@ -0,0 +1,23 @@
+provider "oneview" {
+  ov_username   = var.username
+  ov_password   = var.password
+  ov_endpoint   = var.endpoint
+  ov_sslverify  = var.ssl_enabled
+  ov_apiversion = 2600
```
Comment (nabhajit-ray) [inline data_source.tf:6]: api version is 2800 for 6.10

File: oneview/resource_logical_interconnect_group.go [hunk 2]
```diff
@@ -1817,7 +1821,7 @@ func resourceLogicalInterconnectGroupRead(d *schema.ResourceData, meta interface
 
 	qualityOfService := make([]map[string]interface{}, 0, 1)
 	qualityOfService = append(qualityOfService, map[string]interface{}{
-		"type":                         logicalInterconnectGroup.QosConfiguration.Type,
```
Comment (nabhajit-ray) [inline oneview/resource_logical_interconnect_group.go:1820]: I think gofmt is not running properly . Please run it in local first .

File: data_source.tf (hunk 1 above)
Comment (AsisBagga) [reply data_source.tf:6]: we haven't release for 6.10 yet, hence keeping it to 2600 (OV6.0)

**Review by SHANDCRUZ — APPROVED** [review]

File: oneview/resource_logical_interconnect_group.go (hunk 2 above)
Comment (AsisBagga) [reply oneview/resource_logical_interconnect_group.go:1820]: formatting is done correctly, you are commenting on the removed line.

**Review by nabhajit-ray — APPROVED** [review]

**Pull request merged**